helm/hpcc/templates/NOTES.txt

```diff
@@ -1,6 +1,29 @@
 {{- $defaultImage :=  include "hpcc.imageName" (dict "root" $ "me" dict ) | quote }}
 Thank you for installing the HPCC chart version {{ .Chart.Version }} using image {{ $defaultImage }}
 
+{{/* Gather a list of ephemeral and persistant planes */}}
+{{- $storage := (.Values.storage | default dict) -}}
+{{- $match := dict "ephemeral" (list) "persistant" (list) -}}
+{{- $planes := ($storage.planes | default list) -}}
+{{- $searchLabels := list "data" "dali" "sasha" "dll" "lz" -}}
+{{- range $plane := $planes -}}
+{{- $labels := $plane.labels | default (list "data") -}}
+{{- $tag := (hasKey $plane "storageClass" | ternary "ephemeral" "persistant") }}
+{{- range $label := $labels -}}
+{{- if has $label $searchLabels -}}
```

Comment:
style: we're not consistent at all, but I think it makes it more readable if we indent scopes, e.g. from issuers.yaml :
```
spec:
  {{- if not .root.Values.global.noResourceValidation -}}
  {{- if .me.spec.ca }}
   {{- if .me.spec.ca.secretName }}
    {{- if not (lookup "v1" "Secret" .root.Release.Namespace .me.spec.ca.secretName) }}
      {{- $_ := fail (printf "\n\nUsing a local certificate authority requires a CA certificate stored in the secret named '%s'. To disable mTLS security set \"certificates.enabled=false\". To bypass this validation check set \"global.noResourceValidation=true\". \n" .me.spec.ca.secretName ) -}}
    {{- end }}
   {{- end }}
  {{- end }}
  {{- end }}
```